lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -173,6 +173,7 @@ WasmBytecodeGenerator::GenerateFunction()
     m_currentFunc->wasmInfo = m_funcInfo;
     m_nestedIfLevel = 0;
     m_maxArgOutDepth = 0;
+    m_evalStack.Clear();
```

Comment:
Maybe this should be under the WasmLazyTrapPhase flag?

---
In reply to: [66656352](https://github.com/Microsoft/ChakraCore/pull/1117#discussion_r66656352) [](ancestors = 66656352)